File: iPad/VerBank-CSTSv3-IOSClientApi/ClientStation/api/info/InfoOperator_TRADESERV1007.m
//
//  InfoOperator_TRADESERV1007.m
//  VerBank-IOSClientStation
//
//  Created by Allone on 15/8/31.
//  Copyright (c) 2015年 zhanglei. All rights reserved.
//

#import "InfoOperator_TRADESERV1007.h"

#import "Info_TRADESERV1007.h"
#import "API_IDEventCaptain.h"

@implementation InfoOperator_TRADESERV1007

- (void)onInfo:(InfoFather *)info {
    Info_TRADESERV1007 *tinfo = (Info_TRADESERV1007 *)info;
    // 暂时将整个对象传入
    [API_IDEventCaptain fireEventChanged:DATA_ON_MktOrderHasSendToMarket eventData:tinfo];
    
}

@end

File: WeChatService/EX02-CSTSv3-ClientAPI/src/jedi/ex01/client/station/api/info/InfoOperator_TRADESERV1007.java
package jedi.ex01.client.station.api.info;

import jedi.ex01.CSTS3.comm.info.InfoFather;
import jedi.ex01.CSTS3.comm.info.Info_TRADESERV1007;
import jedi.ex01.client.station.api.event.API_IDEventCaptain;
import jedi.ex01.client.station.api.event.API_IDEvent_NameInterface;

public class InfoOperator_TRADESERV1007 extends InfoOperator {

	@Override
	public void onInfo(InfoFather _info) {
		Info_TRADESERV1007 info = (Info_TRADESERV1007) _info;
		API_IDEventCaptain.fireEventChanged(API_IDEvent_NameInterface.DATA_ON_MktOrderHasSendToMarket,
				info.getUptradeOperatorID());
	}

}
